lib/google_analytics/analytics_export_service.rb

```diff
@@ -28,8 +29,35 @@ def export_bad_links(data)
                                          upload_source: file.path,
                                          content_type: 'application/octet-stream')
         end
+        Rails.logger.info "A new file has been uploaded for #{Date.today}"
       end
       response
     end
+
+    def delete_previous_uploads(min_number_to_delete: nil)
+      begin
+        min_number_to_delete = 2 unless min_number_to_delete.present?
```

Comment:
It'd be more idiomatic to use `||=` to set this without the explicit `.present?` check, or we could set a default value in the argument list.